src/threema/container.ts

```diff
@@ -711,24 +711,25 @@ class Messages implements threema.Container.Messages {
         return false;
     }
 
-    public bindTemporaryToMessageId(receiver: threema.BaseReceiver, temporaryId: string, messageId: string): boolean {
+    public bindTemporaryToMessageId(
+        receiver: threema.BaseReceiver, temporaryId: string, messageId: string): threema.Message | null {
```

Comment:
Just a note on formatting: If line breaks are inevitable, I prefer a style similar to the one used by rustfmt:

```typescript
public bindTemporaryToMessageId(
    receiver: threema.BaseReceiver,
    temporaryId: string,
    messageId: string,
): threema.Message | null {
```

Main advantage is that it keeps git history clean because a change to a single argument doesn't modify the entire line.

But feel free to ignore this comment, I don't have a strong opinion on this.

Reply:
I do agree though that it should be done consistently. I struggle with that in my own projects and have only found a viable solution for it for C projects where the code is being automatically reformatted with `clang-format`.